XpringKit/ILP/auth/IlpCredentials.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftGRPC
+
+/// A helper which provides a convenient way to initialize grpc.Metadata and
+/// add an Authorization metadata header, and ensures every bearer token
+/// going over the wire is prefixed with "Bearer "
+class IlpCredentials {
```

Comment:
Any reason to expose this? Should we label this as internal?

Reply:
No, yes